operator/src/network/controller.rs

```diff
@@ -183,12 +186,34 @@ async fn reconcile(
         });
     };
 
-    let datadog: DataDogConfig = (&spec.datadog).into();
+    // Check if the network should die, otherwise update ttl_seconds_remaining.
+    let creation_timestamp = network.meta().creation_timestamp.as_ref();
+    status.ttl_seconds_remaining = if let Some(ttl_seconds) = &spec.ttl_seconds {
+        if let Some(created) = creation_timestamp {
+            let now = Utc::now();
+            let elapsed = now - created.0;
+            let ttl_seconds_remaining = elapsed.num_seconds() - ttl_seconds;
```

Comment:
isn't this arithmetic backwards?  Shouldn't it be tll - elapsed, not elapsed - ttl?